src/main/java/core/basesyntax/service/ParserService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface ParserService<T> {
+    List<T> parser(List<String> inputData);
```

Comment:
```suggestion
    List<T> parse(List<String> inputData);
```

Methods are usually named a verb.